plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -0,0 +1,33 @@
+defaults:
+  - hydra/sweeper: ax
+
+# Banana function
+# https://en.wikipedia.org/wiki/Rosenbrock_function
+banana:
+  # To be minimized
+  x: ???
+  y: ???
+  
+
+hydra:
+  ax:
```

Comment:
Can you confirm indicate that the hydra.ax node is optional?
(Test it and add a comment saying it).

Reply:
This is the canonical example.
Add a comment saying the hydra.ax node is optional otherwise everyone will always copy paste it.

Reply:
How does this sound: `# The following part of config is used to setup the Hydra Ax plugin and is optional`

Reply:
A bit wordy :)

Reply:
```yaml
hydra:
  # optional Ax specific configuration
  ax:
    ..
```